releases/v20.2.14.md

```diff
@@ -0,0 +1,110 @@
+- Correlated subqueries that couldn't be decorrelated and that have their own subqueries are now executed correctly when supported. Note that this is a very rare edge case. [#67571][#67571]
+- Fixed a very rare unexpected error from the [vectorized engine](../v20.2/vectorized-execution.html) ("index out of bounds") when evaluating the `CASE` operator. [#67785][#67785]
+- Catching up [Raft](../v20.2/architecture/replication-layer.html#raft) followers on the Raft log is now more efficient in the presence of many large Raft log entries. This helps avoid situations where Raft leaders struggle to retain leadership while catching up their followers. [#67126][#67126]
+- Fixed a rare bug in v20.2 that would happen when a lease change occured during a range merge. This bug allowed [`AS OF SYSTEM TIME`](..//v20.2/as-of-system-time.html) queries to serve inconsistent results. [#65823][#65823]
+- The [v20.2 SQL shell](../v20.2/cockroach-sql.html) now properly reports detailed execution timings (using the client-side configuration setting `verbose_times`) when run against a v21.1+ server. [#67853][#67853]
+- Fixed a deadlock that could occur when many replicas were rapidly queued for removal. [#65861][#65861]
+- Fixed two bugs which affected [geospatial queries](../v20.2/spatial-features.html) with the [`st_distance` function](../v20.2/functions-and-operators.html#spatial-functions). The first caused errors for filters of the form `st_distance(g1, g2, use_spheroid) = 0`. The second incorrectly transformed filters of the form `st_distance(g1, g2) = 0` when `g1` and `g2` are geographies to `st_instersects(g1, g2)`. This is not a valid transformation because `st_distance` makes spheroid-based calculations by default while `st_intersects` only makes sphere-based calculations. [#67412][#67412]
+- Fixed an issue with statistics estimation in the [optimizer](../v20.2/cost-based-optimizer.html) that could have caused it to over-estimate the number of rows for some expressions and thus choose a sub-optimal plan. This issue could happen when multi-column statistics were used in combination with histograms, the query contained a predicate on two or more columns where the columns were hightly correlated, and the selected values were very common according to the histograms. [#68012][#68012]
+- Fixed an issue where CockroachDB could encounter an internal error or crash when performing a cast of a `NULL` [`JSON` value](../v20.2/jsonb.html) to [Geography or Geometry types](../v20.2/spatial-features.html). [#67903][#67903]
+- [`INSERT`](../v20.2/insert.html) and [`UPDATE`](../v20.2/update.html) statements which operate on larger rows are now split into batches using the `sql.mutations.mutation_batch_byte_size` setting. [#67963][#67963]
+- Fixed a rare bug that could result in a crash while creating a [`debug.zip`](../v20.2/cockroach-debug-zip.html). The bug was only possible to hit if a `debug.zip` was captured during a period of rapid lease movement. [#67727][#67727]
```

Comment:
The wording in the second sentence caught my ear while reading, maybe something like "It was only possible to hit the bug if...." or "... encounter the bug if..."